changed.d

```diff
@@ -101,7 +101,11 @@ auto getIssues(string revRange)
     foreach (repo; ["dmd", "druntime", "phobos", "dlang.org", "tools", "installer"]
              .map!(r => buildPath("..", r)))
     {
-        auto cmd = ["git", "-C", repo, "fetch", "upstream", "--tags"];
+        // ensure that changed_upstream exists (we ignore failures)
+        auto cmd = ["git", "-C", repo, "remote", "add", "changed_upstream", "https://github.com/dlang/" ~ repo.baseName];
+        execute(cmd);
```

Comment:
We can fetch directly without adding a remote:

```
cmd = ["git", "-C", repo, "fetch", "--tags", "https://github.com/dlang/" ~ repo.baseName, "+refs/heads/*:refs/remotes/upstream/*", "--tags"];
```

This avoids adding a remote and preserves compatibility with the existing command-line usage (specifying `upstream/` before branch names), though it will overwrite the tracking refs of whatever remote the user has that's named `upstream`, assuming one exists.

Git namespaces seem to allow fixing everything at once, though.